python/rikai/__version__.py

```diff
@@ -12,4 +12,4 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-version = "0.0.4.dev0"
+version = "0.0.4dev0"
```

Comment:
This seems not match to PEP 440?

https://www.python.org/dev/peps/pep-0440/#id29

Reply:
technically PEP440 does allow for omitting that separator. But yeah, with the period is the "normal" form. I can add that in.

Reply:
added.